Assets/MRTK/Providers/XRSDK/XRSDKDeviceManager.cs

```diff
@@ -43,7 +43,13 @@ public virtual bool CheckCapability(MixedRealityCapability capability)
         protected static readonly Dictionary<InputDevice, GenericXRSDKController> ActiveControllers = new Dictionary<InputDevice, GenericXRSDKController>();
 
         private readonly List<InputDevice> inputDevices = new List<InputDevice>();
+        private readonly List<InputDevice> inputDevicesSubset = new List<InputDevice>();
         private readonly List<InputDevice> lastInputDevices = new List<InputDevice>();
+        protected static List<InputDeviceCharacteristics> DesiredCharacteristics = new List<InputDeviceCharacteristics>()
```

Comment:
Small potential improvement going forward: do we want this to be `virtual` so platform-specific device managers can override it?